airflow/api_connexion/openapi/v1.yaml

```
@@ -4920,6 +4924,30 @@ components:
 
         *New in version 2.1.1*
 
+    FilterUpdatedAtLTE:
+      in: query
+      name: updated_at_lte
+      schema:
+        type: string
+        format: date-time
+      required: false
+      description: |
+        Returns objects less or equal the specified date.
+
+        This can be combined with updated_at_gte parameter to receive only the selected period.
```

```suggestion
        This can be combined with updated_at_gte parameter to receive only the selected period.
        
        *New in version 2.6.0*
```
Same with the one below